website/source/intro/vs/kubernetes.html.md

```diff
@@ -8,9 +8,8 @@ description: |-
 
 # Nomad vs. Kubernetes
 
-Kubernetes is an orchestration system for Docker developed by the Cloud Native
-Computing Foundation (CNCF). Kubernetes aims to provide all the features
-needed to run Docker based applications including cluster management,
+Kubernetes is an orchestration system for containers originally designed by Google, now governed by the Cloud Native
+Computing Foundation (CNCF) and devloped by Google, Red Hat, CoreOS and many others. Kubernetes aims to provide all the features needed to run docker or rkt-based applications including cluster management,
```

Comment:
@dadgar I intentionally left them lowercase so as to attempt to refer to the project names, not company names (in the case of Docker). Yes, I can update.